mqemitter-mongodb.js

```diff
@@ -10,6 +10,10 @@ const pump = require('pump')
 const nextTick = process.nextTick
 const EE = require('events').EventEmitter
 
+function toStream (op) {
```

Comment:
Used to keep back compatibility with mongo driver < 4

Reply:
Do we need to?

Reply:
@mcollina The problem is when users are passing an instance of mongodb in emitter constructor, it could be an old one and in that case it would stop working... Giving that supporting it is really easy using this help function I think we could release a major with this and remove it on next major release